### PR TITLE
Upgrade Gradle 6.1.1

### PR DESCRIPTION
**Pull request by KeithYokoma**
We have many stuff from the previous version :)

### CODE REVIEW

**Comment by kittinunf** [comment]
🤔 The CI is failing though ...

**Comment by KeithYokoma** [comment]
I think we need to change the way to access properties in `gradle.properties` 🤔 . Something like https://github.com/mercari/RxReduxK/pull/25 seems to be more robust.

**Comment by kittinunf** [comment]
Yes, go ahead! :)

**Comment by kittinunf** [comment]
Sorry ... sorry wrong button

File: buildSrc/src/main/kotlin/Config.kt [hunk 1]
```diff
@@ -11,7 +11,7 @@ object MavenUrl {
 
 object Version {
   const val bintray = "1.8.4"
-  const val kotlin = "1.3.10"
```
Comment (KeithYokoma) [inline buildSrc/src/main/kotlin/Config.kt:14]: This is required for Gradle 6.1.1 running with Kotlin DSL

**Review by yhanada — APPROVED** [review]
🚀

File: buildSrc/src/main/kotlin/Config.kt (hunk 1 above)
Comment (kittinunf) [reply buildSrc/src/main/kotlin/Config.kt:14]: Why don't we bump to the latest? It looks like new version seem to be innocuous.

https://blog.jetbrains.com/kotlin/2019/11/kotlin-1-3-60-released/

**Pull request merged**